Trimble.Modus.Components/Controls/TMInput/MultiLineInput.xaml.cs

```diff
@@ -0,0 +1,66 @@
+namespace Trimble.Modus.Components;
+using Trimble.Modus.Components.Controls.BaseInput;
+
+public partial class MultiLineInput : BaseInput
+{
+    #region Bindable Properties
```

Comment:
When AutoSize is disabled, when I keep typing the current cursor position should be disabled.
![simulator_screenshot_8D02846E-8555-40FD-901D-C39EE5B96C15](https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/8c94278e-84d3-4017-b2a7-bcbf5ad0111e)

Reply:
Looks like this is a known issue in Editor.